styles.css

```diff
@@ -85,3 +85,43 @@ nav {
 .motivation {
   background: var(--coral);
 }
+
+/* styling for table and table heads */
```

Comment:
I think our main content should have some `max-width` so it does not span across the whole screen on larger devices.  It feels strange. Imagine Facebook feed page, its main content is in the middle and the rest on sides. Your focus lies on the main content and if you need to look for something else your eyes don't have to keep rolling from side to side.

```
main {
  max-width: 1200px;
  margin: 0 auto;
}
```

I think it would be best to start styling our page with a **mobile first approach** in mind.
That means that any CSS code is by default considered for small devices and making use of the
`@media` rules for larger screens.
I personally don't like creating a CSS class for every little detail, I'd rather use [CSS selectors](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Selectors) such as `:first-child`, `:nth-child`..

This is an example of how would my implementation look like, but I guess it's a matter of style.

```
/* styling for table and table heads */

#forum-table {
  width: 90%;
  margin: 0 auto;
  margin-top: 25px;
  border-collapse: collapse;
  text-indent: 7px;
}

#forum-table thead tr {
  border-bottom: 1px solid var(--primary-color);
  line-height: 30px;
  font-size: 14px;
  font-weight: 300;
  text-align: left;
}

#forum-table thead tr th:not(:first-child) {
  width: 10%;
  min-width: 30px;
}

#forum-table thead tr .views,
#forum-table thead tr .activity {
  display: none;
}

/* 
  Sizes inspired by https://getbootstrap.com/docs/5.2/layout/breakpoints/

  Anything which is not wrapped by @media is styled for sizes
  from 0px to 767px by default => Mobile first approach   

  Small device >= 576px (576->767) 
  Medium device >= 768px (768->991)
  Large device >= 992px (992->x)
*/

@media (min-width: 768px) {
  #forum-table thead tr .activity {
    display: table-cell;
  }
}

@media (min-width: 992px) {
  #forum-table thead tr .views {
    display: table-cell;
  }
}
```

Reply:
Hey @Dorfieeee!
Thanks for the feedback. Your way of css does look cleaner. I was hoping to know how do we usually make changes in the branch. I mean once I make changes in the branch in vs code how to go about then.

Thanks again.

Reply:
Interesting. I didn't know about `display: table-cell` .